app/frontend/src/features/auth/constants.tsx

```diff
@@ -33,7 +33,7 @@ export const CONTACT =
   "Admins will be happy to change your gender label. Please contact support@couchers.org!";
 export const CURRENT_PASSWORD = "Current password";
 export const ENTER_EMAIL = "Enter your username/email";
-export const FEMALE = "Female";
+export const WOMAN = "A Woman";
```

Comment:
Can this be A_WOMAN so we don't accidentally use it in the wrong place :D

Reply:
Sounds good! I'll give it an update!